public/styles/reset.js

```diff
@@ -0,0 +1,359 @@
+/* http://meyerweb.com/eric/tools/css/reset/
+   v2.0-modified | 20110126
+   License: none (public domain)
+*/
+
+import { createGlobalStyle } from 'styled-components';
+
+const ResetStyles = createGlobalStyle`
+
```

Comment:
I think it should be better to move styles from `reset.js` to `globalStyles.js` with refactoring:
- check duplicate styles
- check tabulation
- remove all comments
- use `::` for pseudo elements